src/resolver.rs

```diff
@@ -2526,6 +2689,18 @@ impl FailForViolationConflict {
             }
         }
 
+        fn print_unaudited_entry(
+            out: &mut dyn Write,
+            entry: &UnauditedDependency,
+        ) -> Result<(), VetError> {
+            writeln!(out, "unaudited {}", entry.version)?;
+            writeln!(out, "      criteria: {:?}", entry.criteria)?;
+            if let Some(notes) = &entry.notes {
+                writeln!(out, "      notes: {notes}")?;
```

Comment:
I wonder a bit if we'd want to put a line break before the notes if they're multiline notes to avoid breaking formatting. Probably a problem for a follow-up.

Reply:
The diagnostic stuff I'm messing with would ideally end up with us producing a Span on the original toml and just printing it out